src/main/java/org/apache/sling/junit/ServiceGetter.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package org.apache.sling.junit.rules;
+package org.apache.sling.junit;
```

Comment:
At first I tried keeping it under the rules package as public, but it didn't seem to work. Maybe it was my mistake, but when I moved it up, my IDE found it. I can try moving it under `org.apache.sling.junit.impl` if we are OK with it being public. (I don't see why this helper can't be public)..  But if it should be protected, then I think it needs to be parent of sub-packages that need to use it.

Reply:
I have no problem with it being public. But I would prefer not to export it (as in OSGi's Export-Package). I.e. it should not be official API but instead remain an implementation detail. That's why I suggest the `impl` package, because - by convention - it is not exported.

Reply:
Yes. This is was done